src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -45,6 +45,13 @@ class DatabaseTokenRepository implements TokenRepositoryInterface
      */
     protected $expires;
 
+    /**
+     * Minimum number of seconds before re-redefining the token.
```

Comment:
"re-redefining". What?

Reply:
Oops, "redefining" of course. Just a second.